src/keysignote.ts

```diff
@@ -15,28 +18,32 @@ export class KeySigNote extends Note {
     this.ignore_ticks = true;
   }
 
-  getBoundingBox() {
+  getBoundingBox(): BoundingBox | undefined {
```

Comment:
I think we can remove this whole method, now?. It just calls the parent.

Reply:
done